student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom, age, name: 'unknown', parent_permission: true)
+    super(id, age, name, parent_permission)
```

Comment:
- [ ] Kindly use the colon notation to assign the `parent_permission` and `name` parameters for the super to avoid ArgumentError as you already use the colon notation in the parent class(Person), so it will be like:
```ruby
super(age, name: name, parent_permission: parent_permission)
```
- [ ] Kindly don't pass the id to the super as it's generated randomly at the parent class, it does not exist in the parameter list of the parent and That variable is not defined.